activerecord-oracle_enhanced-adapter.gemspec

```diff
@@ -82,7 +82,7 @@ This adapter is superset of original ActiveRecord Oracle adapter.
     "spec/spec_helper.rb"
   ]
   s.add_dependency(%q<rspec>, ["~> 3.3"])
-  s.add_dependency(%q<activerecord>, ["= 5.0.0.rc2"])
+  s.add_dependency(%q<activerecord>, ["~> 5"])
```

Comment:
At this moment it is unknown if we wil be able support Rails 5.0 and Rails 5.1 with the same version of Orale enhanced adapter. then how about "~> 5.0" to support only Rails 5.0.